common_features.mk

```diff
@@ -114,8 +114,27 @@ ifeq ($(strip $(RGBLIGHT_ENABLE)), yes)
     endif
 endif
 
-RGB_MATRIX_ENABLE ?= no
```

Comment:
Thanks GitHub.  

For a second there, this really scared me.  But noooooo, it's just how github handles the diff.